aequilibrae/paths/traffic_assignment.py

```diff
@@ -474,12 +480,89 @@ def info(self) -> dict:
                 'Target RGap': self.assignment.rgap_target}
         return info
 
-    def save_skims(self, matrix_name: str, which_ones='final') -> None:
+    def save_skims(self, name: str, which_ones='final', format='omx') -> None:
         """Saves the skims (if any) to the skim folder and registers in the matrix list
 
         Args:
-            matrix_name (:obj:`str`): Name of the file to hold this matrix
+            name (:obj:`str`): Name of the file to hold this matrix
             which_ones (:obj:`str`,optional): {'final': Results of the final iteration, 'blended': Averaged results for
             all iterations, 'all': Saves skims for both the final iteration and the blended ones} Default is 'final'
+            *format* (:obj:`str`, `Optional`): File format ('aem' or 'omx'). Default is 'omx'
         """
-        pass
+
+        mat_format = format.lower()
+        if mat_format not in ['omx', 'aem']:
+            raise ValueError('Matrix needs to be either OMX or native AequilibraE')
+        if mat_format == 'omx' and not has_omx:
+            raise ImportError('OpenMatrix is not available on your system')
+
+        file_name = f'{name}.{mat_format}'
+
+        mats = Matrices()
+        export_name = path.join(mats.fldr, file_name)
+
+        if path.isfile(export_name):
+            raise FileExistsError(f'{file_name} already exists. Choose a different name or matrix format')
+
+        if mats.check_exists(name):
+            raise FileExistsError(f'{name} already exists. Choose a different name')
+
+        avg_skims = self.classes[0].results.skims  # type: AequilibraeMatrix
+
+        # The ones for the last iteration are here
+        last_skims = self.classes[0]._aon_results.skims  # type: AequilibraeMatrix
+
+        names = []
+        if which_ones in ['final', 'all']:
+            if last_skims.names:
+                for core in last_skims.names:
+                    names.append(f'{core}_final')
+
+        if which_ones in ['blended', 'all']:
+            if avg_skims.names:
+                for core in avg_skims.names:
+                    names.append(f'{core}_blended')
```

Comment:
```suggestion
        if which_ones in ['blended', 'all'] and avg_skims.names:
            for core in avg_skims.names:
                names.append(f'{core}_blended')
```

Reply:
I fixed in a more comprehensive way, where the skims object is always an instance of an AequilibraE matrix and it might just be an empty one